os/docker/Dockerfile

```diff
@@ -0,0 +1,33 @@
+ARG ARCH=amd64
+FROM ghcr.io/faddat/archlinux-$ARCH
```

Comment:
after the first build, we can change this image path to:

ghcr.io/gnolang/os

We need to bootstrap from somewhere, so this url needs to stay because we will use github actions for builds